docs/contextvars.rst

```diff
@@ -61,3 +62,23 @@ The general flow is:
    >>> clear_contextvars()
    >>> log.msg("hi there")
    event='hi there' a=None
+
+
+If e.g. your request handler calls a helper function that needs to
+temporarily override some contextvars before restoring them back to
+their original values, you can use the :class:`~contextvars.contextvars.Token`\s
+returned by :func:`~structlog.contextvars.bind_contextvars` along with
+:func:`~structlog.contextvars.reset_contextvars` to accomplish this
+(much like how :meth:`contextvars.ContextVar.reset` works):
```

Comment:
```suggestion
If e.g. your request handler calls a helper function that needs to temporarily override some contextvars before restoring them back to their original values, you can use the :class:`~contextvars.contextvars.Token`\s returned by :func:`~structlog.contextvars.bind_contextvars` along with :func:`~structlog.contextvars.reset_contextvars` to accomplish this (much like how :meth:`contextvars.ContextVar.reset` works):
```
We use [semantic newlines](http://rhodesmill.org/brandon/2012/one-sentence-per-line/).

Reply:
Done in the latest revision.

(Though, as a fan of semantic line breaks myself, I was surprised to get this suggestion. I think adding additional line breaks at appropriate positions within a single sentence is actually encouraged, not forbidden. See the example at https://sembr.org starting with "We can further clarify the source text by adding a line break after the clause". This makes reading and editing docs (and diffs) easier in editors that are not set to soft-wrap long lines, as well as pagers and other tools.)